build.gradle

```diff
@@ -26,7 +26,7 @@ ext {
     bfdVersion='2.1.4'
     aggregatorVersion='1.3.4'
     filtersVersion='1.9.4'
-    eventClientVersion='1.12.7'
+    eventClientVersion='1.12.8'
```

Comment:
I'm not sure, you need to increase the version. All your changes are in src/test/

Reply:
Reverted this change. I think you are right.